test/e2e/fixtures.go

```diff
@@ -189,6 +189,7 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, bool, bool, error) {
 	}
 	if testOptions.providerName == "kind" {
 		// In Kind cluster, there are issues with DNS name resolution on worker nodes.
+		// We will skip TLS testing for Kind cluster because the server certificate is generated with Flow aggregator's DNS name
```

Comment:
As part of Flow Aggregator windows support #2138, we are supporting cluster IP in the certificate. Can we enable the kind test with TLS?

Reply:
Sure. updated